src/types/api-types.ts

```diff
@@ -78,7 +78,7 @@ export interface Breakpoint {
   variableTable: Array<Variable|null>;
   // TODO: The `controller.ts` file assumes `id` is not null or undefined.
   //       Verify it it should be optional.
-  id?: string;
+  id?: BreakpointId;
```

Comment:
You can actually make this non-optional and remove the TODO. Looking at the [docs](https://cloud.google.com/debugger/api/reference/rest/v2/debugger.debuggees.breakpoints) each of the endpoints that yield `Breakpoint` objects, every description states that `id` is guaranteed to be set.

This way, you can also get rid of the `as string` casts elsewhere.

Reply:
Done.